skll/config.py

```diff
@@ -684,8 +684,6 @@ def _parse_config_file(config_path, log_level=logging.INFO):
     # make sure all the specified paths/files exist
     train_path = _locate_file(train_path, config_dir)
     test_path = _locate_file(test_path, config_dir)
-    train_file = _locate_file(train_file, config_dir)
```

Comment:
Are these not used?

Reply:
The variables `train_file` and `test_file` are not used again. This is because they get set to `train_path`/`test_path` right above this point (and those are the values that get returned - it's the logic around standardizing the file vs. directory issue). However, I see now upon closer inspection that `_locate_file` is checking the existence of the file, thereby catching that issue here and not later. So, these lines actually should not be removed. I think they can just be replaced with the method calls by themselves, though, since the return value is not used.

Reply:
Actually, the lines right above (for `train_path`/`test_path`) already do the checking. `train_path`'s value gets set to `train_file`'s value right above this, so, it's just duplicating it.